### ISSUE TITLE
Trouble saving tracks to the playlist

### ISSUE THREAD

**Issue by 536b6f74746b69** [plamere/SortYourMusic#55]
When trying to save or overwrite a playlist an error dialog pops up saying: "Trouble saving tracks to the playlist"

In javascript console there are CORS related errors showing:

Access to XMLHttpRequest at 'http://smarterplaylists.playlistmachinery.com/logger/log?log=sort|spotify:playlist:xxxxxxxxxxx|artist%20separation|asc' from origin 'http://sortyourmusic.playlistmachinery.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by D0miH** [comment]
Yes, same problem for me. A fix would be nice. I love this website!!! :D

**Comment by zeus1921** [comment]
maybe something about the SSL.
Yea please fix it, i love it too!

**Comment by trentharms** [comment]
@plamere - just tagging for visibility 🙏 I love this tool and miss it dearly

**Comment by michaelk1017** [comment]
Same Problem :(

**Comment by huonglarne** [comment]
@plamere We are dying for the tool to be running again. I would really appreciate if you can fix it. Thank you.

**Comment by huonglarne** [comment]
https://github.com/plamere/SortYourMusic/pull/54

**Comment by huonglarne** [comment]
The problem is fixed in this PR https://github.com/plamere/SortYourMusic/pull/54. Hope @plamere can merge it! Thank you

**Comment by largesprite** [comment]
@plamere can you fix this already?? people have let you know about this issue for months.

**Comment by bjornswe** [comment]
@sonneveld Is it possible for you to merge the fix? We are many who love this project but are not able to use it anymore

**Comment by StarshipGnomes** [comment]
@plamere & @sonneveld, we'd really appreciate your assist on reviewing and resolving this. 

> The problem is fixed in this [PR #54](https://github.com/plamere/SortYourMusic/pull/54).

Thank you both for this awesome tool! I've been missing it the past few weeks.

**Comment by 536b6f74746b69** [comment]
fixed with latest update
